packages/plugin-commands-env/src/env.ts

```diff
@@ -98,6 +102,46 @@ export async function handler (opts: NvmNodeCommandOptions, params: string[]) {
     return `Node.js ${nodeVersion as string} is activated
   ${dest} -> ${src}`
   }
+  case 'uninstall': {
+    if (!opts.global) {
+      throw new PnpmError('NOT_IMPLEMENTED_YET', '"pnpm env use <version>" can only be used with the "--global" option currently')
+    }
+
+    const fetch = createFetchFromRegistry(opts)
+    const { releaseChannel, versionSpecifier } = parseNodeEditionSpecifier(params[1])
+    const nodeMirrorBaseUrl = getNodeMirror(opts.rawConfig, releaseChannel)
+    const nodeVersion = await resolveNodeVersion(fetch, versionSpecifier, nodeMirrorBaseUrl)
+    const nodeDir = getNodeVersionsBaseDir(opts.pnpmHomeDir)
+
+    if (!nodeVersion) {
+      throw new PnpmError('COULD_NOT_RESOLVE_NODEJS', `Couldn't find Node.js version matching ${params[1]}`)
+    }
+
+    const versionDir = path.resolve(nodeDir, nodeVersion)
+
+    if (!existsSync(versionDir)) {
+      throw new PnpmError('ENV_NO_NODE_DIRECTORY', `Couldn't find Node.js directory in ${versionDir}`)
+    }
+
+    const nodePath = path.resolve(opts.pnpmHomeDir, 'node')
+    const nodeLink = await fs.readlink(nodePath).catch(() => '')
+
+    if (nodeLink.includes(versionDir)) {
+      const npmPath = path.resolve(opts.pnpmHomeDir, 'npm')
+      const npxPath = path.resolve(opts.pnpmHomeDir, 'npx')
+
+      await Promise.all([
+        await fs.unlink(nodePath),
+        await fs.unlink(npmPath),
+        await fs.unlink(npxPath),
```

Comment:
there's no need to await these calls if you paths the to Promise.all
```suggestion
        fs.unlink(nodePath),
        fs.unlink(npmPath),
        fs.unlink(npxPath),
```